tests/splines/matrix.cpp

```diff
@@ -48,14 +46,14 @@ void check_inverse(ddc::DSpan2D matrix, ddc::DSpan2D inv)
         for (std::size_t j(0); j < N; ++j) {
             double id_val = 0.0;
             for (std::size_t k(0); k < N; ++k) {
-                id_val += matrix(i, k) * inv(j, k);
+                id_val += matrix(i, k) * inv(k, j);
```

Comment:
Was it a silent bug ?

Reply:
For example if you look there: https://github.com/CExA-project/ddc/blob/417bd9004fba6ccf1f2f35ee195666146178fe35/include/ddc/kernels/splines/matrix.hpp#L70